elftools/dwarf/ranges.py

```diff
@@ -16,19 +16,22 @@
 
 RangeEntry = namedtuple('RangeEntry', 'entry_offset entry_length begin_offset end_offset is_absolute')
 BaseAddressEntry = namedtuple('BaseAddressEntry', 'entry_offset base_address')
+# If we ever see a list with a base entry at the end, there will be an error that entry_length is not a field.
 
-def not_implemented(e):
-    raise NotImplementedError("Range list entry %s is not supported yet" % (e.entry_type,))
+def _translate_startx_length(e, cu):
+    # Avoid retrieving the start offset twice - I don't think Python can do inline assignments
```

Comment:
This comment can be removed, IMHO. It's fairly clear what's going on

Reply:
Removed.